File: Math/CPP/count_odd_numbers_in_an_interval_range.cpp
class Solution {
    public:
        int countOdds(int l, int h) {
            int nums = h - l + 1, hn = nums / 2;
            bool isOdd = nums & 0x1, lOdd = l & 0x1;
            if (isOdd && lOdd) return hn + 1;
            return hn;
        }
    };

File: Math/JS/count_odd_numbers_in_an_interval_range.js
/**
 * @param {number} low
 * @param {number} high
 * @return {number}
 */
var countOdds = function(l, h) {
    const nums = h - l + 1;
    const isOdd = nums & 1, lOdd = l & 1;
    const hn = Math.floor(nums / 2)
    if (isOdd && lOdd) return hn + 1;
    return hn
};